compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -1406,6 +1398,21 @@ private void handleDeclaredWithVar(BLangVariable variable) {
         }
     }
 
+    private void handleWildCardBindingVariable(BLangSimpleVariable simpleVariable, BType rhsType) {
+        if (simpleVariable.name.value.equals(Names.IGNORE.value)) {
+            BType expectedType = simpleVariable.getBType();
+            BType actualType = rhsType;
+            if (expectedType != null) {
```

Comment:
When is this null? Do we have tests for the same?

Reply:
This is null at this level when used with `var`
var _ = 10;